sdks/python/apache_beam/runners/interactive/utils.py

```diff
@@ -427,3 +436,30 @@ def create_var_in_main(name: str,
     from apache_beam.runners.interactive import interactive_environment as ie
     ie.current_env().watch({name: value})
   return name, value
+
+
+def assert_bucket_exists(bucket_name):
+  # type: (str) -> None
+
+  """Asserts whether the specified GCS bucket with the name
+  bucket_name exists.
+
+    Logs an error and raises a ValueError if the bucket does not exist.
+
+    Logs a warning if the bucket cannot be verified to exist.
+  """
+  try:
```

Comment:
You can make it something like:
```
try:
  from apitools.base.py.exceptions import HttpError
  storage_client = ...
  ...
  storage_client.buckets.Get(request)
except HttpError as e:
  if ...
  ...
  else:
      _LOGGER.warning(
          'HttpError - unable to verify whether bucket %s exists', bucket_name)
except ImportError as e:
     _LOGGER.warning(
          'ImportError - unable to verify whether bucket %s exists', bucket_name)
```

Reply:
Replied on the other comment - resolved, thanks!